test/unit/Assessment/castVotes.js

```diff
@@ -73,7 +85,7 @@ describe('castVotes', function () {
       await setTime(timestamp + daysToSeconds(1));
     }
 
-    await assessment.connect(user).castVotes([0], [true], 0);
+    await assessment.connect(user).castVotes([0], [true], ['ipfsAssessmentDataHash'], 0);
```

Comment:
consider just making a constant at the top of this test file since this string appears all over